[2021-12-25T21:12:25.790397+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2021-12-25T21:12:25.869079+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:12:26.128879+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5ee712"},"request_uri":"http://localhost:8000/_wdt/5ee712","method":"GET"} []
[2021-12-25T21:19:17.479817+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:19:17.481282+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:19:17.485492+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:19:34.265940+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2021-12-25T21:19:34.268353+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:19:34.376646+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"eb4d7f"},"request_uri":"http://localhost:8000/_wdt/eb4d7f","method":"GET"} []
[2021-12-25T21:19:38.805230+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/index","method":"GET"} []
[2021-12-25T21:19:38.806736+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:19:38.815746+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:19:59.364001+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/index"" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/index\" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index/\". at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2021-12-25T21:19:59.408846+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:19:59.410016+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:19:59.731574+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"7b9e42"},"request_uri":"http://localhost:8000/_wdt/7b9e42","method":"GET"} []
[2021-12-25T21:20:05.703640+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:20:05.705195+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:20:05.714157+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:23:38.951248+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:23:38.952724+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:23:38.972131+05:30] request.CRITICAL: Uncaught PHP Exception ErrorException: "Warning: Undefined variable $number" at /home/itachi/srv/xopun-home/home/src/Controller/HomeController.php line 19 {"exception":"[object] (ErrorException(code: 0): Warning: Undefined variable $number at /home/itachi/srv/xopun-home/home/src/Controller/HomeController.php:19)"} []
[2021-12-25T21:23:39.029167+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:23:39.030601+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:23:39.252062+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8a38d9"},"request_uri":"http://localhost:8000/_wdt/8a38d9","method":"GET"} []
[2021-12-25T21:23:51.628701+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:23:51.630302+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:23:51.636988+05:30] request.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "home/index.html.twig" (looked into: /home/itachi/srv/xopun-home/home/templates, /home/itachi/srv/xopun-home/home/vendor/symfony/twig-bridge/Resources/views/Form)." at /home/itachi/srv/xopun-home/home/vendor/twig/twig/src/Loader/FilesystemLoader.php line 227 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \"home/index.html.twig\" (looked into: /home/itachi/srv/xopun-home/home/templates, /home/itachi/srv/xopun-home/home/vendor/symfony/twig-bridge/Resources/views/Form). at /home/itachi/srv/xopun-home/home/vendor/twig/twig/src/Loader/FilesystemLoader.php:227)"} []
[2021-12-25T21:23:51.665990+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:23:51.667099+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:23:51.892334+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3dcf54"},"request_uri":"http://localhost:8000/_wdt/3dcf54","method":"GET"} []
[2021-12-25T21:24:38.086523+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:24:38.087500+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:24:38.124691+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:28:19.389667+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2021-12-25T21:28:19.392775+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:28:19.495897+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0d5f1d"},"request_uri":"http://localhost:8000/_wdt/0d5f1d","method":"GET"} []
[2021-12-25T21:28:56.962482+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://localhost:8000/\" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:186)"} []
[2021-12-25T21:28:56.964898+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:28:57.110353+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9be987"},"request_uri":"http://localhost:8000/_wdt/9be987","method":"GET"} []
[2021-12-25T21:29:08.734046+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:29:08.735056+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:29:08.739792+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:30:22.139727+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:30:22.140754+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:30:22.145047+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:30:28.941618+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException: "No route found for "GET http://localhost:8000/": Method Not Allowed (Allow: POST)" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 134 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\MethodNotAllowedHttpException(code: 0): No route found for \"GET http://localhost:8000/\": Method Not Allowed (Allow: POST) at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:134)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\MethodNotAllowedException(code: 0):  at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:46)"} []
[2021-12-25T21:30:28.976295+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:30:28.977382+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:30:29.239140+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"002b4e"},"request_uri":"http://localhost:8000/_wdt/002b4e","method":"GET"} []
[2021-12-25T21:36:51.550630+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:36:51.551639+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:36:51.556897+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:40:18.501325+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:40:18.503051+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:40:18.515677+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:40:19.374497+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:40:19.375301+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:40:19.381832+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:41:05.871097+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:41:05.874298+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:41:05.890760+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:41:05.986900+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0ff152"},"request_uri":"http://localhost:8000/_wdt/0ff152","method":"GET"} []
[2021-12-25T21:41:08.716278+05:30] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"0ff152"},"request_uri":"http://localhost:8000/_profiler/0ff152?panel=logger","method":"GET"} []
[2021-12-25T21:41:29.287375+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:41:29.290101+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:41:29.312194+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:41:29.408765+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f28edf"},"request_uri":"http://localhost:8000/_wdt/f28edf","method":"GET"} []
[2021-12-25T21:41:55.803841+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:41:55.806487+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:41:55.825229+05:30] request.CRITICAL: Uncaught PHP Exception Twig\Error\SyntaxError: "Unexpected character "$"." at /home/itachi/srv/xopun-home/home/templates/home/index.html.twig line 6 {"exception":"[object] (Twig\\Error\\SyntaxError(code: 0): Unexpected character \"$\". at /home/itachi/srv/xopun-home/home/templates/home/index.html.twig:6)"} []
[2021-12-25T21:41:55.874676+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:41:55.876177+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:41:56.139045+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"7b110c"},"request_uri":"http://localhost:8000/_wdt/7b110c","method":"GET"} []
[2021-12-25T21:42:01.600105+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:42:01.602983+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:42:01.621274+05:30] request.CRITICAL: Uncaught PHP Exception Twig\Error\SyntaxError: "Unknown "number" tag." at /home/itachi/srv/xopun-home/home/templates/home/index.html.twig line 6 {"exception":"[object] (Twig\\Error\\SyntaxError(code: 0): Unknown \"number\" tag. at /home/itachi/srv/xopun-home/home/templates/home/index.html.twig:6)"} []
[2021-12-25T21:42:01.672368+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:42:01.673434+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:42:01.915502+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"740fc3"},"request_uri":"http://localhost:8000/_wdt/740fc3","method":"GET"} []
[2021-12-25T21:42:11.284259+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:42:11.287026+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:42:11.316596+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:42:11.446116+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c48049"},"request_uri":"http://localhost:8000/_wdt/c48049","method":"GET"} []
[2021-12-25T21:42:34.151751+05:30] request.INFO: Matched route "app_home_index". {"route":"app_home_index","route_parameters":{"_route":"app_home_index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://localhost:8000/","method":"GET"} []
[2021-12-25T21:42:34.154585+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-25T21:42:34.173199+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-25T21:42:34.289460+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3e5dae"},"request_uri":"http://localhost:8000/_wdt/3e5dae","method":"GET"} []
[2021-12-25T21:42:40.097843+05:30] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"3e5dae"},"request_uri":"http://localhost:8000/_profiler/3e5dae?panel=logger","method":"GET"} []
[2021-12-25T21:43:05.238032+05:30] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"3e5dae"},"request_uri":"http://localhost:8000/_profiler/3e5dae?panel=events","method":"GET"} []
[2021-12-25T21:43:06.192554+05:30] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"3e5dae"},"request_uri":"http://localhost:8000/_profiler/3e5dae?panel=router","method":"GET"} []
[2021-12-25T21:43:31.442009+05:30] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"3e5dae"},"request_uri":"http://localhost:8000/_profiler/3e5dae?panel=router","method":"GET"} []
[2021-12-26T09:25:49.489282+05:30] request.INFO: Matched route "home.index". {"route":"home.index","route_parameters":{"_route":"home.index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://dev.symfony.xopun.com/","method":"GET"} []
[2021-12-26T09:25:49.514166+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-26T09:25:49.701096+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-26T09:25:49.915666+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"19bfb2"},"request_uri":"http://dev.symfony.xopun.com/_wdt/19bfb2","method":"GET"} []
[2021-12-26T09:25:49.982419+05:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://dev.symfony.xopun.com/favicon.ico" (from "http://dev.symfony.xopun.com/")" at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php line 130 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET http://dev.symfony.xopun.com/favicon.ico\" (from \"http://dev.symfony.xopun.com/\") at /home/itachi/srv/xopun-home/home/vendor/symfony/http-kernel/EventListener/RouterListener.php:130)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /home/itachi/srv/xopun-home/home/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:74)"} []
[2021-12-26T09:25:50.114266+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-26T09:25:50.115392+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-26T09:43:28.975521+05:30] request.INFO: Matched route "home.index". {"route":"home.index","route_parameters":{"_route":"home.index","_controller":"App\\Controller\\HomeController::index"},"request_uri":"http://dev.symfony.xopun.com/","method":"GET"} []
[2021-12-26T09:43:28.977187+05:30] security.DEBUG: Checking for authenticator support. {"firewall_name":"main","authenticators":0} []
[2021-12-26T09:43:28.989831+05:30] app.ERROR: error while trying to collect executed migrations {"exception":"[object] (Doctrine\\DBAL\\Exception\\DriverException(code: 0): An exception occurred in the driver: could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:91)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:26)\n[previous exception] [object] (PDOException(code: 0): could not find driver at /home/itachi/srv/xopun-home/home/vendor/doctrine/dbal/src/Driver/PDO/PgSQL/Driver.php:28)"} []
[2021-12-26T09:43:29.630620+05:30] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"fbfcd5"},"request_uri":"http://dev.symfony.xopun.com/_wdt/fbfcd5","method":"GET"} []
